### DISCUSSION TITLE
Thoughts on a slight change to the README

### DISCUSSION THREAD

**Discussion by SecondThundeR** [archlinux/archinstall#306] [Ideas]
# "Problem"
Due to a new GitHub feature from March that automatically generates a table of contents for README files, there is an empty item in the current README's table of contents that directs to the icon anchor, which looks a little strange.
### Example:
<div align="center">

![image](https://user-images.githubusercontent.com/36604233/114865988-b0977480-9dfb-11eb-9f6d-812987258eb5.png)
***Here is the icon's anchor***
</div>

<div align="center">

![image](https://user-images.githubusercontent.com/36604233/114866000-b3926500-9dfb-11eb-8b7b-76da1036697f.png)
***Here is an empty item due to empty header***
</div>

# Solution
At the moment I see two workarounds:
- **"Low Effort" Method** - Involves disabling the ToC in the repository settings
- **"Beautification" Method** - Edit the README to make the table of contents look normal

The second method can be done by removing anchor from the icon and adding a main header with the project name
### Example:
<div align="center">

![image](https://user-images.githubusercontent.com/36604233/114868494-a5921380-9dfe-11eb-91a5-482e2d9b30f7.png)
***Here are removed anchor from icon and added header***
</div>

<div align="center">

![image](https://user-images.githubusercontent.com/36604233/114868506-a9259a80-9dfe-11eb-8d66-65818bd12dcd.png)
***Table of Contents now looks fine***
</div>

We can also add a center alignment that looks like this:
<div align="center">

![image](https://user-images.githubusercontent.com/36604233/114869388-93fd3b80-9dff-11eb-9d23-307cd60331df.png)
</div>

In my personal experience, many large repositories prefer to do centering, but here it is up to everyone's personal preference.

# Conclusion

This "problem" does not affect the work with the repository, but from a visual point of view, when opening the ToC, an empty item looks a little strange and can attract attention. Is the README worth these changes, or is it preferable to leave it as it is? *(If so, I can make a PR with these changes)*

**Comment by Torxed** [comment]
I would very much welcome this change.
As long as the content remains roughly the same I wouldn't mind a PR! :D

I'm torn on the center vs non centered logo so maybe hold off on that one?
My brain says yes! Which usually means do the opposite when it comes to designs hehe.

**Reply by SecondThundeR** [reply]
I can comment out the part for the centered logo, so when the time comes for this decision, you can simply uncomment it and logo will be centered

**Reply by Torxed** [reply]
That sounds like magic to my ears! No rush but I'll be looking forward to that PR!
